docs/zh_CN/config/README.md

```diff
@@ -267,15 +267,15 @@ MINIO_ETCD_COMMENT          (sentence)  optionally add a comment to this setting
 
 数据使用情况采集器默认是启用的，通过Envs可以设置更多的交错延迟。
 
-采集器能适应系统速度，并在系统负载时完全暂停。 可以调整采集器的速度，从而达到延迟更新的效果。 每次采集操作之间的延迟都可以通过环境变量`MINIO_DISK_USAGE_CRAWL_DELAY`来调整。 默认情况下，该值为10。 这意味着采集每次操作都将休眠*10x*的时间。
+采集器能适应系统速度，并在系统负载时完全暂停。 可以调整采集器的速度，从而达到延迟更新的效果。 每次采集操作之间的延迟都可以通过环境变量`MINIO_DISK_USAGE_SCANNER_DELAY`来调整。 默认情况下，该值为10。 这意味着采集每次操作都将休眠*10x*的时间。
 
 大多数设置要让采集器足够慢，这样不会影响整体的系统性能。
-设置 `MINIO_DISK_USAGE_CRAWL_DELAY` 为一个 *较低* 的值可以让采集器更快，并且设置为0的时候，可以让采集器全速运行（不推荐）。 设置一个较高的值可以让采集器变慢，进一步减少资源的消耗。
+设置 `MINIO_DISK_USAGE_SCANNER_DELAY` 为一个 *较低* 的值可以让采集器更快，并且设置为0的时候，可以让采集器全速运行（不推荐）。 设置一个较高的值可以让采集器变慢，进一步减少资源的消耗。
 
 示例: 如下设置将使采集器的速度降低三倍, 减少了系统资源的使用，但是反映到更新的延迟会增加。
 
 ```sh
-export MINIO_DISK_USAGE_CRAWL_DELAY=30
+export MINIO_DISK_USAGE_SCANNER_DELAY=30
```

Comment:
Just seeing this, but if we are renaming, lets take out the usage part, since it isn't just for usage any more.
```suggestion
export MINIO_SCANNER_DELAY=30
```